docs/methods.rst

```diff
@@ -114,6 +114,39 @@ To continue a run, specify the ``mcwalkerset`` element before your VMC/DMC block
 
 In the project id section, make sure that the series number is different from any existing ones to avoid overwriting them.
 
+.. _transition_guide:
+
+Transition guide for batched drivers
+------------------------------------
+Under the Exascale Computing Project effort, we developed a new set of QMC drivers, called "batched drivers",
+with a unique feature of batching walkers on demand. Available drivers are ``vmc_batch``, ``dmc_batch`` and ``linear_batch``.
+A new concept "crowd" is introduced as a suborganization of walker population in batched drivers.
+Walkers within a crowd operate their computation in lock-step, which helps the GPU efficiency.
+Walkers between crowds remain fully asynchronous unless operations involving the full population are needed.
+With this batching capability, new drivers enable feature implementations potentially to maximize the performance of a given hardware.
+For OpenMP GPU offload users, batched drivers are musts to effectively use GPUs.
+Keep in mind that batched drivers are GPU agnostic even though they incorporate the need of GPU computing.
```

Comment:
Does "GPU agnostic" mean "doesn't care which brand of GPU - Nvidia, AMD, or Intel"?  Or does it mean "doesn't care if using the GPU or CPU"?

Reply:
Will make unambiguous